numpy/core/tests/test_ufunc.py

```diff
@@ -980,7 +980,7 @@ def test_can_ignore_signature(self):
         assert_array_equal(out, mm_row_col_vec.squeeze())
 
     def test_matrix_multiply(self):
-        self.compare_matrix_multiply_results(np.long)
+        self.compare_matrix_multiply_results(np.compat.long)
```

Comment:
maybe a more specific dtype would be better than using a builtin?

Reply:
How about I make a follow-up that uses parametrize for this test?

Reply:
ok